### PR TITLE
Sentiment analysis demo page

### PR DESCRIPTION
**Pull request by chriskonstad**
No pressure on reviewing this, this is more of an example than a feature to fully merge in.

### CODE REVIEW

File: app/controllers/sent_controller.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  def check # display the form
+  end
+
+  def sent # handle the POST, display the result
+    # do stuff
+    analyzer = Analyzer.new
+    tweet = params["tweet"]
+    puts "TWEET: " + tweet
+    @result = analyzer.process(tweet)
+    render 'check'
```
Comment (bmwwoo) [inline app/controllers/sent_controller.rb]: "double quotes"

Comment (chriskonstad) [inline app/controllers/sent_controller.rb]: yeah true. my bad. Just making a demo page, not really aiming for prod code, but a very valid statement.

**Comment by bmwwoo** [comment]
"Overall I like it but it needs tweaking." (0.1448 probability). Some lines that run through there are positive but return low to mid scores but if the sentimentalizer is just for this class we don't need to care that much.

**Comment by chriskonstad** [comment]
It's the most popular sentimentalizer gem for ruby, so.... unless you want to write a new machine learning sentimentalizer, we're stuck with it.  Also, we can give it more data for training.

**Pull request closed without merging**